src/Settings/Flavours/Common.hs

```diff
@@ -0,0 +1,11 @@
+module Settings.Flavours.Common where
```

Comment:
Shall we have a module dedicated to various temporary hacks and workarounds? It could live at the top `src/ToDo.hs`, and it will be the go-to place for Hadrian maintainers for introducing and -- importantly -- finding forgotten temporary fixes that can now be removed?

Reply:
I hesitated, but imagine it gets empty, we delete it? And then we have new hacks, we recreate it? So I just went with the very obvious `Common`, where we can put anything that's shared between flavours, hacks or not :)

Reply:
OK, let's stick with your naming. I agree, this module may be useful for other common flavour-related settings.

Reply:
OK great. I'll let you review everything now, I addressed your other bit of feedback.